pkg/backup/util/util.go

```diff
@@ -455,3 +474,17 @@ func ParseImage(image string) (string, string) {
 	}
 	return name, tag
 }
+
+// canSkipSetGCLifeTime returns if setting tikv_gc_life_time can be skipped based on the TiKV version
+func canSkipSetGCLifeTime(image string) bool {
+	_, version := ParseImage(image)
+	v, err := semver.NewVersion(version)
+	if err != nil {
+		klog.Errorf("Parse version %s failure, error: %v", version, err)
+		return false
```

Comment:
`latest`, `nightly` are invalid Semantic Version, I think we can return true here.